annotations/firefox_desktop_background_update/README.md

```diff
@@ -0,0 +1,5 @@
+As of this writing, the Firefox background updater is Windows-only.
+
+Firefox itself also sends an update ping when an update is available and after one has been applied (on all platforms).
```

Comment:
I don't know if I've written on this anywhere, but style (at least mine) is to quote ping names whenever used.

```suggestion
Firefox itself also sends an "update" ping when an update is available and after one has been applied (on all platforms).
```